cpp/tests/rolling/grouped_rolling_range_test.cpp

```diff
@@ -207,8 +207,7 @@ struct GroupedRollingRangeOrderByNumericTest : public BaseGroupedRollingRangeOrd
     auto const following = make_unbounded_range_bounds();
     auto const results   = get_grouped_range_rolling_sum_result(preceding, following, *order_by);
 
-    auto const expected_results = bigints_column{{6, 6, 6, 6, 6, 6, 8, 8, 8, 8, 12, 12, 12, 12},
-                                                 cudf::test::iterators::no_nulls()};
```

Comment:
Why were all the `no_nulls` removed? I guess because it's the default?

Reply:
This is a slight change in behaviour, in that rolling_window didn't always remove the validity mask, while `groupby::aggregate()` and `cudf::reduce()` do.